packaging/fedora/snapd.spec

```diff
@@ -750,6 +788,10 @@ popd
 %{_datadir}/selinux/devel/include/contrib/snappy.if
 %endif
 
+%if %{with apparmor}
+%config %{_sysconfdir}/apparmor.d/usr.libexec.snapd.snap-confine
```

Comment:
This behavior seems strange? Do we want the file to be overwritten by default with user local changes being backed up and no longer in effect? Also, this line needs to be in the `%files -n snap-confine` section, not the `%files selinux` subpackage section.